Package.swift

```diff
@@ -40,14 +32,15 @@ let package = Package(
             exclude: [
                 "CMakeLists.txt"
             ],
-            swiftSettings: markdownSwiftSettings
+            swiftSettings: [.unsafeFlags(["-Xcc", "-DCMARK_GFM_STATIC_DEFINE"], .when(platforms: [.windows]))]
```

Comment:
Should all platforms have this flag? What does it do for the platforms that didn't need it?

Reply:
This is more of a question for @compnerd than me. I think there's a difference between how packages are linked on Windows and elsewhere that requires that we specify `static` linkage for Windows when building toolchains.

Reply:
On non-Windows platforms, it normally wouldn't have any impact. However, if you were to build with `-fvisibility-default=hidden` it would internalise the symbols, preventing them from participating in dynamic linking.

Reply:
@d-ronnqvist The syntactic difference between `.when(platforms: [.windows])` and `#if os(Windows)` is that the latter restricts the flag to *building from a Windows host*, but the former restricts the flag to *building for a Windows target*. Switching it back to the `.when` form allows us to cross-compile DocC for Windows from other hosts (which is important for toolchain builds) with correct linkage.